configs/ld/ld_gflv1_r101_r18_fpn_coco_1x.py

```diff
@@ -0,0 +1,109 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
```

Comment:
`train_pipeline` seems to be the same with its base config. You may just ignore this key in `data.train` config, and the base config would be adopted.